fppanalysis/two_dim_velocity_estimates.py

```diff
@@ -113,6 +141,13 @@ def estimate_velocities_for_pixel(x, y, ds: xr.Dataset):
     of the maximums of the two cross-correlations involved (good luck
     understanding this last sentence :))
 
+    Input:
+        x: pixel index x
+        y: pixel index y
+        ds: xarray Dataset
+        cross_cond_av = True: Estimates velocity field of cross conditional average
+        cross_cond_av = False: Estimates velocity field of cross correlation 
```

Comment:
Use a string, instead of a bool. I.e. argument should be "method", and have possible values "cond_av" and "cross_corr". Raise and exception otherwise.